### PR TITLE
feat: add path explore

### PR DESCRIPTION
**Pull request by tyyzqmf**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

(describe what this merge request does)

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [ ] add new test cases
- [ ] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

File: frontend/src/apis/analytics.ts [hunk 1]
```diff
@@ -30,42 +31,47 @@ export const createAnalyticsDashboard = async (
...
   projectId: string;
   appId: string;
+  attribute?: boolean;
 }) => {
-  const result: any = await apiRequest(
-    'get',
-    `/metadata/events?projectId=${params.projectId}&appId=${params.appId}`
-  );
+  const { attribute } = params;
+  let path = `/metadata/events?projectId=${params.projectId}&appId=${params.appId}`;
```
Comment (zxkane) [inline frontend/src/apis/analytics.ts:70]: Why not define API path as `/metadata/${projectId}/${appId}/events` to make the consistent API sytle?

Comment (chenhaiyun) [inline frontend/src/apis/analytics.ts:70]: When the get parameters is long , we can try this:
> const url = new URL('/metadata/events');
> const paramsObj = new URLSearchParams({
>   projectId,
>   appId,
> });
> 
> if (attribute) {
>   paramsObj.append('attribute', attribute);
> }
> 
> url.search = paramsObj.toString();
> const path = url.pathname + url.search;

**Review by zxkane — CHANGES_REQUESTED** [review]

File: frontend/src/apis/analytics.ts (hunk 1 above)
Comment (tyyzqmf) [reply frontend/src/apis/analytics.ts:70]: track #215

**Review by zxkane — CHANGES_REQUESTED** [review]
resolve conflicts locally

**Review by zxkane — APPROVED** [review]

**Pull request merged**